proposals/0296-increase-tx-size.md

```diff
@@ -0,0 +1,177 @@
+---
+simd: '0296'
+title: Increase Transaction Size
+authors:
+  - jacobcreech
+  - apfitzge
+category: Standard
+type: Core
+status: Review
+created: 2025-05-28
+feature: (fill in with feature key and github tracking issues once accepted)
+---
+
+## Summary
+
+Historically, messages transmitted over the network must not exceed the IPc6 MTU
+size to ensure a fast and reliable network delivery. Solana has used a 
+conservative MTU size of 1280 bytes, which after accounting for some of the
+overhead, leaves a maximum transaction size of 1032 bytes for data in the
+payload.
+
+With QUIC, we can now use a larger transaction size than the original 1280
+bytes. This proposal presents a new transaction size starting at 4096 bytes,
```

Comment:
Since this transaction format aims to obviate the need for address lookup tables, and address lookup tables can today hold 256 32-byte addresses, isn't 4096 bytes too low to achieve the goal?

Reply:
4096 bytes achieves the goal for now until max account load increases from 64. There's still the case where if you have an address be repeated many times over a ALT will be more efficient on packing today, but I don't see much usage like that in the wild.